### PR TITLE
Fix build and support for celery==5 and django==3.1

### PR DESCRIPTION
**Pull request by arnau126**
- Update flake8 to 3.8.3 to fix
`AttributeError: 'FlakesChecker' object has no attribute 'CONSTANT'`

- Remove HOST and  PORT from DATABASES to fix
`django.db.utils.OperationalError: fe_sendauth: no password supplied`

- Remove "-U" options from `pip install celery` to fix
`ModuleNotFoundError: No module named 'celery.five'`

- Add `databases = '__all__'` in test_Models to fix
`AssertionError: Database queries to 'secondary' are not allowed in this test. Add 'secondary' to t.unit.test_models.test_Models.databases to ensure proper test isolation and silence this failure.`

- Modify docstring of TaskResultManager.store_result function to fix:
`D417: Missing argument descriptions in the docstring (argument(s) meta, traceback are missing descriptions in 'store_result' docstring)`

### CODE REVIEW

**Comment by arnau126** [comment]
I would appreciate a quick merge, as this PR will fix test errors from other PRs. @thedrow @auvipy

**Comment by arnau126** [comment]
I've added another commit: "Add support for Celery==5 and Django==3.1".

File: tox.ini [hunk 1]
```diff
@@ -33,8 +35,8 @@ deps=
 sitepackages = False
 recreate = True
 commands =
-    pip install -U celery
-    pip install -U kombu
+    pip install celery
```
Comment (Surgo) [inline tox.ini:38]: Why remove `-U` option?
I think it is not safe 🤔 
All package and cached and cannot find backward incompatible changes.
https://github.com/celery/django-celery-results/blob/v1.0.4/.travis.yml#L3

How about `pip install -r requirements/default.txt` ?

Comment (Surgo) [reply tox.ini:38]: Ooops :(
`-r{toxinidir}/requirements/default.txt` is defined in `deps`

I think that installations are unnecessary

**Review by auvipy — APPROVED** [review]

**Pull request merged**